static/app/plugins/components/settings.tsx

```diff
@@ -174,21 +176,21 @@ class PluginSettings<
       }
       return (
         <div className="m-b-1">
-          <div className="alert alert-warning m-b-1">{data.config_error}</div>
-          <a className="btn btn-primary" href={authUrl}>
+          <Alert type="warning">{data.config_error}</Alert>
+          <Button priority="primary" href={authUrl}>
```

Comment:
does this need to be marked as external

Reply:
While it generally is an external site we want the user to end up back where they started when the account linking process is complete and `external` button links open a new tab.